networkx.gemspec

```diff
@@ -21,7 +21,9 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   spec.add_development_dependency 'bundler', '~> 1.15'
+  spec.add_development_dependency 'nmatrix'
```

Comment:
`nmatrix` and `rb_heap` are runtime dependencies, not development dependencies

Reply:
Ah, I'm not well versed with gemspec. Sorry for this.. :slightly_frowning_face: